integration/network/network_test.go

```diff
@@ -103,3 +107,41 @@ func TestNetworkList(t *testing.T) {
 		})
 	}
 }
+
+func TestAPINetworkGetDefaults(t *testing.T) {
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	defaults := []string{"bridge", "host", "none"}
+	if runtime.GOOS == "windows" {
```

Comment:
Probably more correct would be to check the daemon's platform. We don't currently do so, but these tests _could_ be run from a Linux environment testing a Windows daemon;


```go
	if testEnv.DaemonInfo.OSType == "windows" {
```